### PR TITLE
Replace TravisCI and AppVeyor with GH Actions and fix deprecation of `body` keyword

### PR DESCRIPTION
**Pull request by PetarKirov**
Before this change, when a program uses `influx-d` the following deprecation
message about the `body` keyword [[⁰]][0] would be printed with dmd >= 2.097.0:

    .dub/packages/influx-d-0.4.4/influx-d/source/influxdb/api.d(770,7):
    Deprecation: Usage of the `body` keyword is deprecated. Use `do` instead.

This commit changes that by using the newer expression contract syntax, added in
DMD 2.081.0 [[¹]][1].

[0]: https://dlang.org/changelog/2.097.0.html#body-deprecation
[1]: https://dlang.org/changelog/2.081.0.html#expression-based_contract_syntax

### CODE REVIEW

**Comment by PetarKirov** [comment]
It's seems like TravisCI is failing due to this issue: https://github.com/dlang/dmd/discussions/12123

I suggest replacing them with GitHub Actions. I'll take a look if this is easily doable.

**Comment by PetarKirov** [comment]
Done! It seems to work smoothly on my fork: https://github.com/PetarKirov/influx-d/actions/runs/1472229633

image

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: CI Pipeline
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
+    name: Unit tests
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```
Comment (PetarKirov) [inline .github/workflows/ci.yml:9]: > macOS-latest workflows will use macOS-11 soon. For more details, see https://github.com/actions/virtual-environments/issues/4060

@atilaneves @John-Colvin shall I replace `macOS-latest` with `macOS-10.15`? Given https://issues.dlang.org/show_bug.cgi?id=21919 I think only dmd-nightly would work on macOS 11 (as of https://github.com/dlang/druntime/pull/3612).

**Comment by PetarKirov** [comment]
GitHub Actions build for 75254f5:
https://github.com/PetarKirov/influx-d/actions/runs/1472387376

**Comment by atilaneves** [comment]
Please delete the appveyor.yml file as well.

**Comment by PetarKirov** [comment]
@atilaneves why do you think I didn't? See: https://github.com/symmetryinvestments/influx-d/pull/51/commits/a92db996ff848ed1565c865cd78a287a25eb7e40.

**Comment by PetarKirov** [comment]
@atilaneves can I haz some tag pls

**Pull request merged**